test/fixtures/scale.time/source-auto-series.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+	threshold: 0.01,
+	config: {
+		type: 'line',
+		data: {
+			labels: ['2017', '2018', '2019', '2020', '2025'],
```

Comment:
It seems weird that this mode creates ticks for 2021 -> 2024 as well whereas other tests like `source-data-series-offset-min-max` do not

Reply:
That seemed weird to me at first too. But `source: 'data'` is actually same as `source: 'labels'` when the data is primitive (the other axis value comes from labels).
So it actually makes sense, there is no data points between 2020 and 2025, so no ticks.